core/Dispatch.cpp

```diff
@@ -31,38 +32,96 @@ namespace olympia
 
         auto execution_topology = coreutils::getExecutionTopology(node->getParent());
 
-        // Create Disptchers for ALU, FPU, BR -- one to many of them
-        // depending on the execution_topology extension
-        for (auto exe_unit_pair : execution_topology)
+        auto issue_queue_pipe_topology = olympia::coreutils::getPipeTopology(node->getParent(), "issue_queue_topology");
+
+        for (size_t i = 0; i < issue_queue_pipe_topology.size(); i++)
         {
-            const auto tgt_name   = exe_unit_pair[0];
-            const auto unit_count = exe_unit_pair[1];
+            auto exe_units = issue_queue_pipe_topology[i];
+            auto exe_unit_first = exe_units[0];
```

Comment:
Why drop the `const`?

Reply:
Resolved in [6b0e363](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/6b0e3633c1b7331170afda8920f596f9fd16b233)